### PR TITLE
Logging added

### PR DESCRIPTION
**Pull request by youtweakit**
Sample playbook for local install. Using pip installed Ansible 2.4.2.0 on Ubuntu Xenial (16.04 latest).
`parity-home` logs can be found in `home` folder
`parity-foreign` logs can be found in `foreign` folder

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -1,4 +1,40 @@
 ---
+### default user is root(change if you want to run from other sudoer)
```
Comment (phahulin) [inline group_vars/all.yml]: This should go to hosts.yml

File: hosts.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+HostsIP ansible_connection=local
```
Comment (phahulin) [inline hosts.yml]: This is not a valid yml, also it should not be commited to repository as it is specific to individual deployment

Comment (youtweakit) [reply hosts.yml]: okay. i see maybe localhost naming? )

Comment (phahulin) [reply hosts.yml]: What's `localhost naming`?

Comment (youtweakit) [reply hosts.yml]: HostIP renaming to something like "localhost"

File: roles/preconfig/templates/poa-logrotate.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/home/{{ ansible_user }}/logs/*.log {
```
Comment (phahulin) [inline roles/preconfig/templates/poa-logrotate.conf.j2]: there is no such folder, it should be `{{ base_path }}/{{ home_name }}/logs`. Also, there should be two blocks of patterns: one for `home` and for `foreign`

File: roles/parity-foreign/tasks/main.yml [hunk 4]
```diff
@@ -40,6 +48,13 @@
   tags:
     - configs
 
+- name: Add log file in node.toml
```
Comment (phahulin) [inline roles/parity-foreign/tasks/main.yml]: Better move this to `node.tom.j2` template

File: hosts.yml (hunk 2 above)
Comment (phahulin) [reply hosts.yml]: I mean, this file should not be in the repository in the first place. Different people will use different IP address, change it and then they won't be able to merge latest changes

File: roles/preconfig/templates/poa-logrotate.j2 [hunk 5]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+/usr/sbin/logrotate /home/{{ username }}/poa-logrotate.conf
```
Comment (phahulin) [inline roles/preconfig/templates/poa-logrotate.j2]: `username` is undefined

File: roles/parity-home/tasks/main.yml [hunk 6]
```diff
@@ -18,6 +18,14 @@
     path: "{{ base_path }}/{{ home_name }}/parity_data/keys/{{ home_chain_name }}"
     state: directory
 
+- name: Create logs directories
+  file:
+    path: "/{{ base_path }}/{{ home_name }}/logs/old"
+    state: directory
+    mode: 0755
+    owner: "{{ ansible_user }}"
```
Comment (phahulin) [inline roles/parity-home/tasks/main.yml]: `ansible_user` will be owner of this by default

File: roles/preconfig/logrotate.yml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+---
+- name: Install poa-logrotate conf
+  template: src=poa-logrotate.conf.j2 dest={base_path}/poa-logrotate.conf owner=root group=root mode=0644
```
Comment (phahulin) [inline roles/preconfig/logrotate.yml]: should be in double brackets `{{ ... }}`

File: roles/parity-home/tasks/main.yml (hunk 6 above)
Comment (youtweakit) [reply roles/parity-home/tasks/main.yml]: I think that it's not a problem... ansible_user is a sudoer or root by default

File: hosts.yml (hunk 2 above)
Comment (youtweakit) [reply hosts.yml]: oh yes, you right, i'll rename it to example-hosts

File: hosts-example-local-install.yml [hunk 8]
```diff
@@ -0,0 +1,2 @@
+---
+HostsIP ansible_connection=local
```
Comment (phahulin) [inline hosts-example-local-install.yml]: Still, it's not a valid yml... why do you need this file anyway?

File: roles/parity-home/tasks/main.yml [hunk 9]
```diff
@@ -18,6 +18,14 @@
     path: "{{ base_path }}/{{ home_name }}/parity_data/keys/{{ home_chain_name }}"
     state: directory
 
+- name: Create logs directories
+  file:
+    path: "/{{ base_path }}/{{ home_name }}/logs/old"
+    state: directory
+    mode: 0755
+    owner: root
```
Comment (phahulin) [inline roles/parity-home/tasks/main.yml]: why root?

File: roles/parity-foreign/tasks/main.yml [hunk 10]
```diff
@@ -18,6 +18,14 @@
     path: "{{ base_path }}/{{ foreign_name }}/parity_data/keys/{{ foreign_chain_name }}"
     state: directory
 
+- name: Create logs directories
+  file:
+    path: "{{ base_path }}/{{ foreign_name }}/logs/old"
+    state: directory
+    mode: 0755
+    owner: "{{ ansible_user }}"
```
Comment (phahulin) [inline roles/parity-foreign/tasks/main.yml]: it will belong to `ansible_user` anyway

File: group_vars/all.yml (hunk 1 above)
Comment (youtweakit) [reply group_vars/all.yml]: That's about comment? or ansible_user: root?

Comment (phahulin) [reply group_vars/all.yml]: That's about this whole file. the content you added should be in hosts.yml

File: hosts-example-local-install [hunk 11]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (phahulin) [inline hosts-example-local-install]: This file should be removed, it's not in yml format and may confuse users reading the README

Comment (youtweakit) [reply hosts-example-local-install]: ok

**Pull request closed without merging**